### ISSUE TITLE
Telemetry data sent via Bluetooth.

### ISSUE THREAD

**Issue by Matorra** [FrSkyRC/ETHOS-Feedback-Community#4626]
I have analyzed the data sent via Bluetooth from the telemetry and I see that no data is sent from the calculated sensors, nor can data from channels or logic switches be sent. In addition, it can also be seen that a lot of data is sent with a very high update frequency that might not be necessary in external applications. I wonder if it is planned to have the possibility of configuring what can be sent via Bluetooth even if it is planned to be able to send analog or binary data from the system itself as I mentioned before.

**Comment by mawzthefinn** [comment]
BT telemetry is just an s.port mirror. So it can’t carry channel or LS data inherently (not supported by the protocol) and it doesn’t get calculated sensors either (as they do not arrive via the RF telemetry connection)

**Comment by Matorra** [comment]
If you had noticed that the values ​​do not always match those seen on the screen and if it has been designed as a mirror of the bus it seems to make sense. Thanks for your information.
If you allow me one more question, is there any official document of the protocol used for this bus?

**Comment by mawzthefinn** [comment]
Yes, documentation for S.port is available under FrSky’s developer program. There’s a link to it on the main website

**Comment by Matorra** [comment]
Thank you mawzthefinn

**Comment by mawzthefinn** [comment]
Closing, answer provided
